### PR TITLE
display quote from api

### PR DESCRIPTION
**Pull request by Paul-tes**
- Used [API Ningas](https://api-ninjas.com/api/quotes) to fech quotes.
- Added Quote Component to display Qutes.
- Added States functionality (loadding + error ) with the functionality of fetching data.

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { useState } from 'react';
 import Result from './Result';
 import Keys from './keys';
 import calculate from '../logic/calculate';
+import Quote from './Ouote';
```
Comment (TadesseDev) [inline src/components/Calculator.jsx]: - [Optional] Dear @Paul-tes, I believe you have misspelled the `Quote`  component into `Ouote`. As professional programmers, we need to be curious about our naming standards. I would suggest naming your component `Quotes.jsx`.

File: src/components/Ouote.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <p className="q-p">
+          {data.quote}
+          {' '}
+          by&apos;
+          {data.author}
+          &apos;
+        </p>
+      </div>
+      <p className="status">{status}</p>
+    </div>
```
Comment (TadesseDev) [inline src/components/Ouote.jsx]: - [x] One main thing missing is, currently you are not displaying any message on the calculator page. All I can see is the `status` and `Created: by"` text. Please make sure to render the fetched quote once it's ready. 
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Paul-tes 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitFlow  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] Please try to separate your code into different levels of commits. Remember your commits are going to help you look back and debug your code. It's a good idea to commit after every visible change.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.jsx (hunk 1 above)
Comment (Paul-tes) [reply src/components/Calculator.jsx]: thanks

File: src/components/Calculator.jsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { useState } from 'react';
 import Result from './Result';
 import Keys from './keys';
 import calculate from '../logic/calculate';
+import Quote from './Quote';
```
Comment (BenMukebo) [inline src/components/Calculator.jsx:5]: - [OPTIOANL] Great job so far 👏🏻 However, The `<Quotes>` component should be imported into the main component which is the App component, and not the Calculator. please change this. 💯 👍🏻 
- Your code should look like this:

```jsx
import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      <Calculator />
      <Quotes />
    </>
  )
}

export default App
```

File: src/components/Quote.jsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from 'react';
+import fetchQuote from '../API/quoteApi';
+
+const Quote = () => {
+  const [data, setData] = useState({
+    quote: '',
+    author: '',
+    category: '',
+  });
+  const [status, setStatus] = useState(null);
```
Comment (BenMukebo) [inline src/components/Quote.jsx:10]: - [OPTIONAL] It's one of the project requirements you create a `loading` state and an `error` state that will handle errors. Please address this:

```jsx
  const [quotes, setQuotes] = useState([])
  const [hasError, setHasError] = useState(false)
  const [isLoading, setIsLoading] = useState(false)
```

**Review by BenMukebo — APPROVED** [review]
### Hi @ , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- The external API [API Ninjas](https://api-ninjas.com/api/quotes) has been used correctly. ✔️  
- The useEffect hook has been used correctly. ✔️  


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**